dashboard/app/models/school.rb

```diff
@@ -21,6 +21,10 @@
 class School < ActiveRecord::Base
   include Seeded
 
+  self.primary_key = 'id'
```

Comment:
I had to add this because I was getting the following error during unit tests: http://stackoverflow.com/questions/18056162/getting-unknown-primary-key-for-table-while-the-id-is-there 

I can't explain why rails isn't auto-detecting the `id` column as primary exactly. It probably has to do with the fact that I didn't let rails auto-create the `id` column, which probably means it never got set as primary (see the other stackoverflow answer).